src/PresentationalComponents/Template/TemplateCard.js

```diff
@@ -39,15 +28,15 @@ TemplateCardHead.propTypes = {
 };
 
 export const TemplateCardActions = ({ children, downloadReport, iconInlineMessage, iconInlineState, ...props }) => (
-    <CardActions { ...props }>
+    <div { ...props }>
```

Comment:
Can this https://www.patternfly.org/components/card#cardheaderactionsobject be replacement of CardActions?

Reply:
This needs to be done, but requires a refactor of 5 different files. I want to keep the refactor of dashboard to a separate PR